### PR TITLE
Add new coredns version 1.5.0

### PR DESCRIPTION
**Pull request by pipo02mix**
Add new coreDNS version 1.4.0

### CODE REVIEW

File: service/controller/aws/v14/version_bundle.go [hunk 1]
```diff
@@ -8,9 +8,9 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "TODO",
-				Description: "TODO",
-				Kind:        versionbundle.KindAdded,
+				Component:   "coredns",
+				Description: "Updated to 1.5.0. More info here: https://coredns.io/2019/04/06/coredns-1.5.0-release/",
```
Comment (rossf7) [inline service/controller/aws/v14/version_bundle.go]: Shouldn't it be 1.5.0 in the PR title too?

On the changelogs is there a breaking change? We got the feedback last time it wasn't visible enough. Should we include here?

Not sure as we will cover in the release notes anyway.

Comment (pipo02mix) [reply service/controller/aws/v14/version_bundle.go]: yes changed

Comment (pipo02mix) [reply service/controller/aws/v14/version_bundle.go]: well It is the changelog of the app repo. Maybe it is good to link here to this changelog. And from there link upstream, WDYT?

Comment (rossf7) [reply service/controller/aws/v14/version_bundle.go]: Yes good call. I think linking to our changelog is good.

Comment (pipo02mix) [reply service/controller/aws/v14/version_bundle.go]: ok done

File: service/controller/aws/v14/version_bundle.go [hunk 2]
```diff
@@ -8,9 +8,9 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "TODO",
-				Description: "TODO",
-				Kind:        versionbundle.KindAdded,
+				Component:   "coredns",
+				Description: "Updated to 1.5.0. More info here: https://github.com/giantswarm/kubernetes-coredns/blob/master/CHANGELOG.md",
```
Comment (rossf7) [inline service/controller/aws/v14/version_bundle.go]: <3

**Pull request merged**